src/main/java/Main.java

```diff
@@ -121,6 +134,31 @@ private void pobierzWynikiPartii() {
 
     }
 
+    private int obliczIloscZdobytychPunktow() {
+        int tab[][] = tab3;
+        int kolumnaStartowa, rob = 0, wiersz = iloscWylicytowanychLew - 1;
```

Comment:
Staraj się wszystkie zmienne deklarować w osobnych linijkach (tyczy się to też innych miejsc w kodzie)